tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -4181,6 +4185,12 @@ TEST_F(ExecutionTest, ATOSampleCmpLevelTest) {
   if (!CreateDevice(&pDevice, D3D_SHADER_MODEL_6_7))
       return;
 
+  if (!DoesDeviceSupportAdvancedTexOps(pDevice)) {
+    WEX::Logging::Log::Comment(L"Device does not support Advanced Texture Operations.");
```

Comment:
Not sure if these get parsed by any automated scripts, but there's minor difference between this text and the previous check ('Ops' vs 'Operations.')

Reply:
It's also a slightly different case since this is not a skip, but just an FYI note that rarely gets read. I'd have changed it for consistency anyway, but I set it to automerge. oh well!